include/RAJA/pattern/forall.hpp

```diff
@@ -475,12 +467,7 @@ RAJA_INLINE void forall(Args&&... args)
 template <typename ExecutionPolicy, typename... Args>
 RAJA_INLINE void forall_Icount(Args&&... args)
 {
-  util::PluginContext context{util::make_context<ExecutionPolicy>()};
-  util::callPreLaunchPlugins(context);
-
   forall_Icount(ExecutionPolicy(), std::forward<Args>(args)...);
```

Comment:
If this calls the above non wrap version of forall wouldnm't this call the plugins more than once?